### PR TITLE
add social links to the footer

### LINKED ISSUE #294: Add social links in footer?
As a nerd, I often like it if I can easily find a project's GitHub and Twitter links somewhere on the page.
I'm thinking adding a GitHub/Twitter links or icons in the site footer is probably easy enough.

### PR DESCRIPTION
**Pull request by johngruen**
fixes #294

### CODE REVIEW

**Comment by ckprice** [comment]
Are we interested in tracking these outbound links? If so, we'll need https://github.com/mozilla/testpilot/issues/1237#issuecomment-240496070

File: legal-copy/legal-templates.js [hunk 1]
```diff
@@ -28,12 +28,17 @@ module.exports.templateEnd = `
...
-          <a data-l10n-id="footerLinkContribute" href="https://www.mozilla.org/contribute/signup/" class="boilerplate">Contribute</a>
+          <div class="legal-links">
+            <a href="https://www.mozilla.org" class="mozilla-logo"></a>
+            <a data-l10n-id="footerLinkLegal" href="https://www.mozilla.org/about/legal/" class="boilerplate">Legal</a>
+            <a data-l10n-id="footerLinkPrivacy" href="/privacy" class="boilerplate">Privacy</a>
+            <a data-l10n-id="footerLinkTerms" href="/terms" class="boilerplate">Terms</a>
+            <a data-l10n-id="footerLinkCookies" href="https://www.mozilla.org/privacy/websites/#cookies" class="boilerplate">Cookies</a>
+          </div>
+          <div class="social-links">
+            <a href="https://github.com/mozilla/testpilot" target="_blank" class="link-icon github"></a>
```
Comment (chuckharmston) [inline legal-copy/legal-templates.js]: We should still have text in the anchor for a11y reasons, even if we use `text-indent` tricks to hide it from traditional user agents.

/g

**Comment by chuckharmston** [comment]
One small note other than @ckprice's comment, otherwise looks good to me. Feel free to merge yourself once you've added the anchor text to links.

**Comment by ckprice** [comment]
FYI checked with Gareth, and we do not get this tracking for free. We'll need to implement it if we want to count these clicks.

**Comment by johngruen** [comment]
@ckprice hey, added GA, can you confirm this is legit-ish? Not sure how i'd do this on the legal pages, but i'm guessing it's not worth the effort.

File: testpilot/frontend/static-src/app/views/footer-view.js [hunk 2]
```diff
@@ -1,12 +1,32 @@
+import app from 'ampersand-app';
 import BaseView from './base-view';
 
 export default BaseView.extend({
   template: `<div id="footer-links">
+                <div class="legal-links">
```
Comment (pdehaan) [inline testpilot/frontend/static-src/app/views/footer-view.js:6]: nit: indenting looks sliiightly off.

File: testpilot/frontend/static-src/app/views/footer-view.js [hunk 3]
```diff
@@ -1,12 +1,32 @@
...
                  <a href="https://www.mozilla.org" class="mozilla-logo"></a>
                  <a data-l10n-id="footerLinkLegal" href="https://www.mozilla.org/about/legal/" class="boilerplate">Legal</a>
                  <a data-l10n-id="footerLinkPrivacy" href="/privacy" class="boilerplate">Privacy</a>
                  <a data-l10n-id="footerLinkTerms" href="/terms" class="boilerplate">Terms</a>
                  <a data-l10n-id="footerLinkCookies" href="https://www.mozilla.org/privacy/websites/#cookies" class="boilerplate">Cookies</a>
-                 <a data-l10n-id="footerLinkContribute" href="https://www.mozilla.org/contribute/signup/" class="boilerplate">Contribute</a>
-             </div>`
+                </div>
+                <div class="social-links">
+                  <a href="https://github.com/mozilla/testpilot" target="_blank" class="link-icon github" title="Github"></a>
```
Comment (pdehaan) [inline testpilot/frontend/static-src/app/views/footer-view.js]: super nit: It's "GitHub"

**Comment by ckprice** [comment]
I see the events coming through! 👍 
events
Just realized that my comment in #1237 mentioned the requirement of `outboundURL`, but that isn't required if we're opening in a new window.

File: testpilot/frontend/static-src/app/views/footer-view.js (hunk 2 above)
Comment (johngruen) [inline testpilot/frontend/static-src/app/views/footer-view.js]: screen shot 2016-08-18 at 4 20 08 pm

@pdehaan wontfix on this b/c like 5 is the one that's off!

**Comment by johngruen** [comment]
@ckprice: merge when it passes?

**Pull request merged**